fortitude/src/settings.rs

```diff
@@ -177,8 +180,19 @@ impl fmt::Display for PreviewMode {
     }
 }
 
-/// Default rule selection
-pub const DEFAULT_SELECTORS: &[RuleSelector] = &[RuleSelector::All];
+lazy_static! {
+    /// Default rule selection
+    pub static ref DEFAULT_SELECTORS: Vec<RuleSelector> = {
```

Comment:
I think it's fine to do this lazily for now. With fewer than 50 default rules, I imagine the performance implications would barely even be measurable.

Reply:
Just timed this: `Built default selectors in: 107.602µs`. Probably fast enough!